File: inbox-outbox-quarkus/integration-tests/src/main/java/pl/messaging/inbox/quarkus/it/model/TestMessageFailing.java
package pl.messaging.inbox.quarkus.it.model;

import java.time.OffsetDateTime;
import java.util.UUID;
import org.jetbrains.annotations.NotNull;
import pl.messaging.model.Message;

public class TestMessageFailing extends Message {
    public TestMessageFailing() {
        super(UUID.randomUUID(), 0, OffsetDateTime.now());
    }

    public TestMessageFailing(@NotNull UUID id) {
        super(id, 0, OffsetDateTime.now());
    }
}

File: inbox-outbox-quarkus/integration-tests/src/main/kotlin/pl/shane3102/messaging/inbox/quarkus/it/model/TestMessageFailing.kt
package pl.shane3102.messaging.inbox.quarkus.it.model;

import java.time.OffsetDateTime;
import java.util.UUID;
import org.jetbrains.annotations.NotNull;
import pl.shane3102.messaging.model.Message;

public class TestMessageFailing extends Message {
    public TestMessageFailing() {
        super(UUID.randomUUID(), 0, OffsetDateTime.now());
    }

    public TestMessageFailing(@NotNull UUID id) {
        super(id, 0, OffsetDateTime.now());
    }
}
